ConsoleApp1/ConsoleApp1/Actions.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsoleApp1
+{
+	public class Actions
+	{
+		public Result Method1()
```

Comment:
Please add normal method names, for instance:
Method1 -> Info()
Method2 -> Warning()
Method3 -> Error()